drivers/flash/soc_flash_nrf.h

```diff
@@ -29,17 +29,11 @@ struct flash_context {
 
 #ifndef CONFIG_SOC_FLASH_NRF_RADIO_SYNC_NONE
 
-#if defined(CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE)
```

Comment:
Why this patch changes timeout both for full-page-erase and partial-page-erase?

Reply:
The thinking is that with `CONFIG_SOC_FLASH_NRF_TIMEOUT_MULTIPLIER`, the two become the same.

I agree with [your comment](https://github.com/zephyrproject-rtos/zephyr/pull/50531/files#r978249130) above that to maintain current behaviour we should change the default for `CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE` though.